bot/slack/commands/ip_info.py

```diff
@@ -0,0 +1,49 @@
+import netaddr
+import os
+
+from mozdef_util.geo_ip import GeoIP
+
+
+def is_ip(ip):
+    try:
+        netaddr.IPNetwork(ip)
+        return True
+    except Exception:
+        return False
+
+
+def ipLocation(ip):
+    location = ""
+    try:
+        geoip_data_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), "../../../data/GeoLite2-City.mmdb")
+        geoip = GeoIP(geoip_data_dir)
+        geoDict = geoip.lookup_ip(ip)
```

Comment:
I think we should really avoid falling into the habit of mixing `snake_case` and `camelCase` as per [PEP 8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names).

Reply:
Fixed in https://github.com/mozilla/MozDef/pull/1018/commits/ffb0a3ca5e9c0525e89a8745c064f7273a5a4036